src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -195,7 +195,7 @@ private static string GenerateUpgradeReport(string projectName, string backupPat
         /// <summary>
         /// Perform the multi-package update action.
         /// </summary>
-        /// <remarks>This needs to be called from a background thread. It may hang on the UI thread.</remarks>
+        /// <remarks>This needs to be called from a background thread. It may make the UI thread stop responding.</remarks>
```

Comment:
freeze?

Reply:
I will follow with current guidance to keep consistency.